ddR/NAMESPACE

```diff
@@ -40,7 +39,7 @@ export(is.dlist)
 export(is.dobject)
 export(is.sparse_darray)
 export(nparts)
-export(parallel)
+export(parallel.ddR)
```

Comment:
I don't really have any major objection to changing the name to this, but we have to agree to consistency. If we follow this convention, this means that backends should always be referred to with a ".ddR" suffix, e.g., one would have to type `useBackend(distributedR.ddR)` for Distributed R. I think this is slightly more cumbersome to type than just `useBackend(distributedR)`. Thoughts?